runtime/src/in_mem_accounts_index.rs

```diff
@@ -491,6 +513,7 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         let entry = map.entry(pubkey);
         m.stop();
         let mut new_entry_zero_lamports = false;
```

Comment:
Probably could get the `is_zero_lamport()` answer here for the `new_entry` if `PreAllocatedAccountMapEntry` had a fn/trait fn for it. Dunno if this is starting to get into over-engineering territory though.